packages/node-core/src/integrations/http/SentryHttpInstrumentation.ts

```diff
@@ -168,20 +214,23 @@ export class SentryHttpInstrumentation extends InstrumentationBase<SentryHttpIns
       // In this case, `http.client.response.finish` is not triggered
       subscribe('http.client.request.error', onHttpClientRequestError);
 
+      if (this.getConfig().createSpansForOutgoingRequests) {
```

Comment:
With the `span` option, we should also check for `this.getConfig().spans` (if I understood that correctly).